src/renderer/components/commands-publish-button.tsx

```diff
@@ -158,28 +159,55 @@ export class PublishButton extends React.Component<PublishButtonProps> {
     );
 
     return (
-      <fieldset disabled={isPublishing}>
-        <ButtonGroup className='button-publish'>
-            <Popover
-              content={privacyMenu}
-              position={Position.BOTTOM}
-            >
+      <>
+        <fieldset disabled={isPublishing}>
+          <ButtonGroup className='button-publish'>
+              <Popover
```

Comment:
This button currently doesn't do anything when updating. We should either:
* Not show the privacy button when updating
* Update the privacy state as well

Reply:
Updating the privacy of gist is not available from API, so I just removed that button when we have gist 🙃